.github/scripts/build.sh

```diff
@@ -7,4 +7,6 @@ if [ $PROJECT = 'escrow' ]; then
     forc build --path $PROJECT/tests/artifacts/asset
 elif [ $PROJECT = 'multisig-wallet' ]; then
     forc build --path $PROJECT
+elif [ $PROJECT = 'fundraiser' ]; then
```

Comment:
Maybe alphabetize this one to avoid merge conflicts with everyone putting stuff at the end.